patterns/contact-info-locations.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Title: Contact: Info and locations
+ * Slug: twentytwentyfive/contact-info-locations
+ * Keywords: contact, location
+ * Categories: contact
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:heading {"textAlign":"left","align":"full","fontSize":"xx-large"} -->
+	<h2 class="wp-block-heading alignfull has-text-align-left has-xx-large-font-size">How to get in touch with us</h2>
```

Comment:
This is computed at `58px` and the design is `70px`.

Reply:
There is no preset for 70px, I think there needs to be a new issue opened about these sizes that are not in the control, and also address the question you asked about whether it should be fluid.